conda_smithy/configure_feedstock.py

```diff
@@ -546,12 +551,13 @@ def _render_ci_provider(
         for config in configs:
             remove_file(config)
 
-        for platform in platforms:
+        for platform, arch in zip(platforms, archs):
             configs = glob.glob(
                 os.path.join(forge_dir, ".ci_support", "{}_*".format(platform))
             )
             for config in configs:
-                remove_file(config)
+                if config not in _GENERATED_CONFIGS:
+                    remove_file(config)
```

Comment:
Let's just remove the files in `.ci_support` at the beginning of the rendering so that we don't have to keep global variables like this

Reply:
Good idea, and it simplifies the whole deletion weirdness